### PR TITLE
feat: add ongoing events filtering

### PR DESCRIPTION
**Pull request by mrtraser**

### CODE REVIEW

File: src/types/types.ts [hunk 1]
```diff
@@ -738,7 +738,7 @@ export interface EventChangeById extends EventPatch, InternalId {}
 
 export interface EventFilter extends CreatedAndLastUpdatedTimeFilter {
   startTime?: DateRange;
-  endTime?: DateRange;
+  endTime?: DateRange | { isNull: boolean };
```
Comment (polomani) [inline src/types/types.ts]: better to define new type or interface
and then you can add typedoc as well

**Review by polomani — COMMENTED** [review]
they've also added `activeAtTime`, can you make support for that as well?

File: src/types/types.ts (hunk 1 above)
Comment (mrtraser) [reply src/types/types.ts]: smthing like `NullableDateRange`?

Comment (polomani) [reply src/types/types.ts]: `NullableProperty`,  `NullableFilter`, or `NullableFilterProperty` would be more flexible

Comment (mrtraser) [reply src/types/types.ts]: done

File: src/__tests__/resources/events.integration.spec.ts [hunk 2]
```diff
@@ -192,6 +192,17 @@ describe('Events integration test', () => {
...
+
+    test('activeAtTime', async () => {
+      const { items } = await client.events.list({
+        filter: {
+          activeAtTime: { min: 101 },
+        },
+        limit: 10,
+      });
+
+      expect(items.length).toBe(1);
```
Comment (mrtraser) [inline src/__tests__/resources/events.integration.spec.ts]: what if somebody adds an event to the tenant which will be at the same time range?
is it better to use `toBeGreaterThan(0)`?

Comment (polomani) [inline src/__tests__/resources/events.integration.spec.ts]: .toBeGreaterThan(0) would be enough

Comment (polomani) [reply src/__tests__/resources/events.integration.spec.ts]: in case someone adds some random event

Comment (BugGambit) [reply src/__tests__/resources/events.integration.spec.ts]: What about https://github.com/cognitedata/cognite-sdk-js/pull/365/commits/fa6802afca6a8e5962708b190adfcbf67d972b03 ?

Comment (BugGambit) [reply src/__tests__/resources/events.integration.spec.ts]: Ok. I can do that and revert https://github.com/cognitedata/cognite-sdk-js/pull/365/commits/fa6802afca6a8e5962708b190adfcbf67d972b03 if you want

Comment (polomani) [reply src/__tests__/resources/events.integration.spec.ts]: https://github.com/cognitedata/cognite-sdk-js/commit/fa6802afca6a8e5962708b190adfcbf67d972b03 is fine

Comment (mrtraser) [reply src/__tests__/resources/events.integration.spec.ts]: yep, a much better solution

Comment (polomani) [reply src/__tests__/resources/events.integration.spec.ts]: I just thought you finished)

Comment (BugGambit) [reply src/__tests__/resources/events.integration.spec.ts]: I reverted back. Test failed due to eventual consistency

Comment (BugGambit) [reply src/__tests__/resources/events.integration.spec.ts]: I reverted back. Test failed due to eventual consistency

**Review by polomani — APPROVED** [review]
looks good

but
DO NOT MERGE BEFORE API RELEASE

**Pull request merged**